dependencies/bom/pom.xml

```diff
@@ -365,6 +365,12 @@
         <artifactId>transport-rest-vertx</artifactId>
         <version>${project.version}</version>
       </dependency>
+      <!-- ServiceComb third-extensions -->
+      <dependency>
```

Comment:
1. sort all modules in folder name, do not put in end
2. modify distributions.xml

Reply:
done